python/pickle/PyROOT_writetests.py

```diff
@@ -90,6 +90,25 @@ def test5WriteCustomTypes( self ):
       pickle.dump(  o, self.out1, protocol = 2 )
       cPickle.dump( o, self.out2, protocol = 2 )
 
+   def test6WriteCustomTypes( self ):
+      """Test writing a RooDataSet with weights"""
```

Comment:
If there is a JIRA associated with this, could you also add in this comment the number?